tests/unit/metadata/test_metadata.py

```diff
@@ -1,74 +1,77 @@
 import re
-from unittest.mock import Mock, call, patch
+from unittest.mock import Mock, call, mock_open, patch
 
 import pandas as pd
 import pytest
 
 from sdv.errors import InvalidDataError
 from sdv.metadata.errors import InvalidMetadataError
 from sdv.metadata.metadata import Metadata
+from sdv.metadata.single_table import SingleTableMetadata
 from tests.utils import DataFrameMatcher, get_multi_table_data, get_multi_table_metadata
 
 
-class TestMetadataClass:
-    """Test ``Metadata`` class."""
-
-    def get_multi_table_metadata(self):
-        """Set the tables and relationships for metadata."""
-        metadata = {}
-        metadata['tables'] = {
-            'users': {
-                'columns': {'id': {'sdtype': 'id'}, 'country': {'sdtype': 'categorical'}},
-                'primary_key': 'id',
-            },
-            'payments': {
-                'columns': {
-                    'payment_id': {'sdtype': 'id'},
-                    'user_id': {'sdtype': 'id'},
-                    'date': {'sdtype': 'datetime'},
-                },
-                'primary_key': 'payment_id',
+@pytest.fixture
+def metadata_instance():
```

Comment:
Added this fixture because we had `self.get_multi_table_metadata` and `get_multi_table_metadata` which was very confusing (self returned one instance and the `get_` without self returned another).